addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -175,6 +177,8 @@
 //
 //Tags_MakeBleed          - Adds bleed to target
 //- duration                - Bleed duration
+//Tags_IgnitePlayer		  - Sets target on fire
```

Comment:
Missing a line comment `//` above to make it consistent

Reply:
resolved